src/main/java/com/hermesworld/ais/galapagos/staging/impl/StagingImpl.java

```diff
@@ -220,6 +222,24 @@ private static ApplicableChange checkForCreateTopic(TopicMetadata topic, String
                     newTopic.isSubscriptionApprovalRequired()));
         }
 
+        if (!oldTopic.getProducers().equals(newTopic.getProducers())) {
```

Comment:
This gives a NullPointerException if either oldTopic or newTopic has a null value for the list.